src/main/java/net/pms/dlna/DLNAMediaAudio.java

```diff
@@ -166,7 +166,7 @@ public boolean isAACLC() {
 	/**
 	 * @return True if the audio codec is AAC HE.
 	 */
-	public boolean isAACHE() {
+	public boolean isHEAAC() {
```

Comment:
@Sami32 As I said in the comment below, I think this is because of a merge-error where one method had been made in one PR and the other in another.